editor/code_editor.cpp

```diff
@@ -1119,7 +1119,11 @@ void CodeTextEditor::update_editor_settings() {
 	text_editor->set_code_hint_draw_below(EDITOR_GET("text_editor/completion/put_callhint_tooltip_below_current_line"));
 	code_complete_enabled = EDITOR_GET("text_editor/completion/code_complete_enabled");
 	code_complete_timer->set_wait_time(EDITOR_GET("text_editor/completion/code_complete_delay"));
-	idle->set_wait_time(EDITOR_GET("text_editor/completion/idle_parse_delay"));
+	if (idle_time == 0.0) {
+		idle->set_wait_time(idle_time);
```

Comment:
This will initialize it to 0.
You can do `bool first_time = idle_time == 0.0` and move the condition after the setting is loaded.

Reply:
There we go